### PR TITLE
ubuntu server page: update copy

### PR DESCRIPTION
**Pull request by andesol**
## Done

- Update copy of /server

## QA

- Open demo
- (Or check out this feature branch and run the site using the command `./run serve` or `dotrun`)
- Go to /server
- Compare changes with [copydoc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4)

## Issue / Card

Fixes https://github.com/canonical/workplace-engineering-squad/issues/856

### CODE REVIEW

File: templates/server/index.html [hunk 1]
```diff
@@ -326,11 +326,10 @@ <h2> Ubuntu Advantage for your Infrastructure </h2>
   </div>
   <div class="row">
     <div class="col-6">
-      <p> Ubuntu Advantage for infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry.
-        With OpenStack and Kubernetes support included, Ubuntu Advantage for Infrastructure provides everything you need to future-proof your data centre. </p>
-      <p> UA for Infrastructure, Advanced includes Server, OpenStack and Kubernetes support and is available at Standard (24x5) and Advanced (24x7) response
-        times. </p>
-      <p><a href="/pricing/infra">Learn more about Ubuntu Advantage for Infrastructure&nbsp;&rsaquo;</a></p>
+      <p> Ubuntu Pro offers a single, per-node packaging of the most comprehensive software, hardening and security in the industry.
```
Comment (sowasred2012) [inline templates/server/index.html:329]: @andreu thanks for this - all looks good except the title for this section also needs updating.

Currently it is: "Ubuntu Advantage for your Infrastructure"
and should just be "Ubuntu Pro"

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
LGTM - just one more change before it can be merged

File: templates/server/index.html (hunk 1 above)
Comment (andesol) [reply templates/server/index.html:329]: Done!

**Review by sowasred2012 — APPROVED** [review]
LGTM!

**Pull request merged**